### PR TITLE
Publishers search box

### PR DESCRIPTION
**Pull request by cormachallinanderilinx**
Issue: https://github.com/IATI/ckanext-iati/issues/399

Staging see: https://staging.iatiregistry.org/dashboard/mypublishers-pending?q=cormac&sort=display_name+asc

### CODE REVIEW

File: ckanext/iati/views/dashboard.py [hunk 1]
```diff
@@ -159,16 +159,21 @@ def my_pending_organizations():
     try:
         if not c.user:
             raise NotAuthorized
-        pending_organizations = iati_h.organization_list_pending()
+
+        q = request.params.get(u'q', u'')
+        c.q = q
```
Comment (EricSoroos) [inline ckanext/iati/views/dashboard.py:164]: Do you need this to be in the request context?  It's generally been deprecated, and you are passing this value into the render later as one of the extra vars.

Comment (EricSoroos) [reply ckanext/iati/views/dashboard.py:164]: Actually, I'm not sure which one you're using, bout're referencing `c.` in the template, so the `extra_vars` might not be used at all.

Comment (cormachallinanderilinx) [reply ckanext/iati/views/dashboard.py:164]: ok, so just remove 
`extra_vars["q"] = c.q`

Comment (EricSoroos) [reply ckanext/iati/views/dashboard.py:164]: Maybe, I'm not sure if it's using the c. or the extra_vars version in the template.

Comment (cormachallinanderilinx) [reply ckanext/iati/views/dashboard.py:164]: removed extra_vars and it is still working

**Pull request merged**